sitzverteilung-service/pom.xml

```
@@ -5,13 +5,6 @@
     <modelVersion>4.0.0</modelVersion>
 
 
-    <parent>
-        <groupId>de.muenchen</groupId>
-        <artifactId>muenchen-pom</artifactId>
-        <version>2.1.0</version>
-    </parent>
-
-
     <groupId>de.muenchen.sitzverteilung</groupId>
```

Hier finde ich "sitzverteilung" zu hoch in der Hierarchie.
```suggestion
    <groupId>de.muenchen.wahl.sitzverteilung.backend</groupId>
```
Es gibt nämlich in [artifactory](https://artifactory.muenchen.de/ui/repos/tree/General/mvn-snapshot-local/de/muenchen/wahl) diese Unterkategorie "wahl"